src/legacy/upload/styles.scss

```diff
@@ -0,0 +1,30 @@
+@charset "utf-8";
```

Comment:
Van waar komt al deze styling? vs https://github.com/milieuinfo/webcomponent-vl-ui-upload/blob/master/src/style.scss

Reply:
Die eronder kwam nog van de originele migratie idd, dus heb het weggedaan. Link komt van de originele js-file constructor waar hij in de <style> werd gezet, dus heb hem bij in onze styles file gezet. Icon herinner ik me niet meer, heb het eruit gehaald.